spikeinterface/sorters/kilosort3/kilosort3.py

```diff
@@ -29,6 +30,31 @@ def check_if_installed(kilosort3_path: Union[str, None]):
     else:
         return False
 
+def check_compiled():
```

Comment:
Maybe this should be a class method so that it could be moved to the base class? Then a class variable called `compiled_name = "ks3_compiled"` could be added to all KS versions and called from within the function. What do you think?